queries/java/highlights.scm

```diff
@@ -50,21 +50,21 @@
 "*="
 "/="
 "%="
+"->"
 ] @operator
 
 ; Types
 
-[
 (interface_declaration
- name: (identifier))
+ name: (identifier) @type)
 (class_declaration
- name: (identifier))
+ name: (identifier) @type)
 (enum_declaration
- name: (identifier)
+ name: (identifier) @type)
 (constructor_declaration
- name: (identifier))
-(type_identifier)
-] @type
+ name: (identifier) @type)
```

Comment:
```suggestion
   name: (identifier) @type)
```

Reply:
The original file used 1 space, I have used 1 space everywhere. Should I change it everywhere?

Reply:
Just realised you have an editorconfig, done